markdown/kubernetes_operator_codelab/kubernetes_operator_codelab.md

```diff
@@ -0,0 +1,243 @@
+author: shwetamazumder
+summary:
+id: kubernetes_operator_codelab
+tags:
+categories:
+environments: Web
+status: Published
+feedback link: https://github.com/SolaceDev/solace-dev-codelabs/blob/master/markdown/kubernetes_operator_codelab
+
+# How to deploy Solace Pubsub+ Event Broker with Pubsub+ Event Broker Operator in Kubernetes
+
+## What you'll learn: Overview
+
+Duration: 0:01:00
+
+Hello and Welcome! If you are wondering how to use our Operator , then you are in the right place! 
+
+This codelab will show how to:
+* Setup a kubernetes environment with Kind
+* Install the Solace Pubsub+ Event Broker Operator in kubernetes
+* Deploy a dev instance of Solace Pubsub+ Event Broker using the Operator
+* Test the Pubsub+ Event Broker deployement and login to Pubsub+ Manager
+
+
+<!--### Info Boxes
+Plain Text followed by green & yellow info boxes 
+
+Negative
+: This will appear in a yellow info box.
+
+Positive
+: This will appear in a green info box.
+
+### Bullets
+Plain Text followed by bullets
+* Hello
+* CodeLab
+* World
+
+### Numbered List
+1. List
+1. Using
+1. Numbers
+
+### Add an Image or a GIF -->
+![Pubsub+ Operator](img/Operator_Img1.png)
+<!--![Soly Image Caption](img/soly.gif) -->
+
+## Prerequisites
+
+Duration: 0:05:00
+
+You will need a Kubernetes Cluster to deploy the Operator. There are several options available on Windows, Mac and Linux to setup a Kubernetes Cluster such as, [Docker Desktop](https://docs.docker.com/desktop/kubernetes/) , [Rancher Desktop](https://rancherdesktop.io/), [kubeadm](https://kubernetes.io/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm/), [minikube](https://minikube.sigs.k8s.io/docs/start/) etc.
+
+I will be using [kind](https://kind.sigs.k8s.io/) to create my Kubernetes Cluster for today's tutorial. It creates the cluster which spins up Docker Containers acting as nodes. 
+
+Note: Kind installation is covered in the next step. If you already have a Kubernetes Cluster on your machine then you can directly skip to Step 4: Install Solace Pubsub+ Event Broker Operator.
```

Comment:
Can we add this in a "Positive" styled format as follows
`<aside class="positive"> Kind installation is covered.... </aside>`